sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/ConfigErrorTest.java

```diff
@@ -27,9 +29,16 @@ class ConfigErrorTest {
   @RegisterExtension
   LogCapturer logs = LogCapturer.create().captureForType(GlobalOpenTelemetry.class);
 
+  @BeforeEach
+  void setup() {
+    GlobalOpenTelemetry.resetForTest();
+    GlobalLoggerProvider.resetForTest();
+  }
+
   @Test
   @SetSystemProperty(key = "otel.propagators", value = "cat")
   void invalidPropagator() {
+    // TODO(jack-berg): confirm log warnings go away once exporters are properly shutdown (#5113)
```

Comment:
There's misleading log messages in the test output that look like this:
```
[Test worker] ERROR io.opentelemetry.exporter.internal.grpc.OkHttpGrpcExporter - Failed to export metrics. The request could not be executed. Full error message: executor rejected
    [Test worker] ERROR io.opentelemetry.exporter.internal.grpc.OkHttpGrpcExporter - Failed to export metrics. The request could not be executed. Full error message: executor rejected
```
They can happen when a test like this initializes a periodic metric reader with an exporter, and the exporter is shutdown before the periodic metric reader. When the periodic metric reader is shutdown it flushes to the exporter which has already had its executor threads shutdown. #5113 fixes this by adjusting all exporters to reject exports after being shutdown. This really is the correct behavior trying to export is bound to produce various exceptions after the network resources have been closed.

Reply:
Are you intending to keep the TODO here? Do we want to wait until 5113 is in the can?

Reply:
Removed since #5113 is merged!